src/main/scala/treadle/Driver.scala

```diff
@@ -11,6 +11,7 @@ import scala.collection.mutable
 //scalastyle:off magic.number
 case class TreadleOptions(
   writeVCD:           Boolean = false,
+  enableCoverage:     Boolean = false,
```

Comment:
To avoid mid-release binary compatibility problems, I think you need to move enableCoverage to the end of the
the parameters for this class.